webpack.config.js

```diff
@@ -220,7 +224,11 @@ function getConfig(options = {}) {
 
                 // Webpack defaults:
                 '.js',
-                '.json'
+                '.json',
+
+                // Typescript:
+                '.tsx',
```

Comment:
Is this necessary?

Reply:
I thought it will be useful for components, regarding the fact that they will have an .tsx extension.

Reply:
I don't think we need to add it because ts-loader will handle those files, this is for custom stuff like our `.web.js`

Reply:
👍